lib/log.js

```diff
@@ -209,9 +215,9 @@ const CodingGameServiceLog = new Lang.Class({
     reversedCopyFileEvents: function() {
       // Go through the event log in reverse and yield copy-file
```

Comment:
Yeah .... unfortunately the auto fix mode doesn't touch comments :( I'll fix these now.